### DISCUSSION TITLE
Not working

### DISCUSSION THREAD

**Discussion by mariusnauj** [mariusz-ostoja-swierczynski/tech-controllers#28] [Q&A]
Hello I Have Tech EU-L-4 Wi FI controler v.1.1.10. I did all steps but when I go to Configuration -> Integrations and click Add button and searching tech it not find. What could be the problem? Maybe with firmware v.1.1.10 integration not working ?

Thank you

**Comment by ondrej-vereb** [comment]
Hi mariusnauj,
I had the same issue. Check your logs in Configuration -> Logs. I found there a message which was saying _"The custom integration 'tech' does not have a version key in the manifest file and was blocked from loading"_. To fix it just add version statement in _/config/custom_components/tech/manifest.json_ .

Mine looks like this:

 {
  "domain": "tech",
  "name": "Tech Controllers",
  "version": "1.0",
  "config_flow": true,
  "documentation": "https://www.home-assistant.io/integrations/tech",
  "requirements": ["tech"],
  "dependencies": [],
  "codeowners": [
    "@mariusz.ostoja-swierczynski"
  ]
}

Regards
Ondrej
